docs/src/index.md

```diff
@@ -525,11 +525,111 @@ To reference columns with more complicated expressions, you must wrap column ref
 @transform df :a + $(get_column_name(x))
 ```
 
-## Constructing multi-column arguments and `src => fun => dest` calls using `$`
+### Operations with multiple columns at once using `AsTable` inside operations
 
-If an argument is entirely wrapped in `$()`, the result bypasses the anonymous function creation of DataFramesMeta.jl and is passed to the underling DataFrames.jl function directly. Importantly, this allows for multi-column selection and passing `src => fun => dest` calls from the DataFrames.jl "mini-language" directly.
+In operations, it is also allowed to use `AsTable(cols)` to work with
+multiple columns at once, where the columns are grouped together in a
+`NamedTuple`. When `AsTable(cols)` appears in a operation, no
+other columns may be referenced in the block.
 
-### Multi-argument selectors 
+`AsTable` on the right-hand-side also allows the use of the special
+column selectors `Not`, `Between`, and regular expressions. As well
+as working with lists of variables programmatically. 
+
+For example, consider a collection of column names `vars`, such that
+
+```
+df = DataFrame(a = [11, 14], b = [17, 10], c = [12, 5]);
+vars = ["a", "b"];
+```
+
+To make a new column which is the sum of `vars`, write
+
+```
+julia> @rtransform df :y = sum(AsTable(vars))
```

Comment:
maybe also add an example when just a column selector is passed to `AsTable` as it will be maybe more common?

Reply:
Added. And another example about how function can now "see" the names of columns
.